client-tez/src/main/java/org/apache/tez/runtime/library/output/RssUnorderedKVOutput.java

```diff
@@ -116,6 +116,8 @@ public RssUnorderedKVOutput(OutputContext outputContext, int numPhysicalOutputs)
   public List<Event> initialize() throws Exception {
     this.startTime = System.nanoTime();
```

Comment:
Why do we use `System.nanoTime()` instead of `System.currentMills()`? @lifeSo

Reply:
Should we fix it in this PR? 

@lifeSo Is startTime used for metrics? We fix it in https://github.com/apache/incubator-uniffle/issues/972?

Reply:
OK.